src/semgrep_agent/semgrep.py

```diff
@@ -390,11 +390,15 @@ def _update_baseline_findings(
                         "--skip-unknown-extensions",
                         "--disable-nosem",
                         "--json",
-                        "--metrics",
-                        "off",  # only count one semgrep run per semgrep-agent run
                         *extra_args,
                         *config_args,
                     ]
+
+                    # Can only disable metrics if auto is not a passed config
+                    # disable so only one metrics run per semgrep-action run
```

Comment:
```suggestion
                    # If possible, disable metrics so that we get metrics only once per semgrep-action run
                    # However, if run with config auto we must allow metrics to be sent
```